src/Core/Framework/Api/Sync/SyncServiceInterface.php

```diff
@@ -11,5 +11,5 @@ interface SyncServiceInterface
      *
      * @throws \Doctrine\DBAL\ConnectionException
      */
-    public function sync(array $operations, Context $context, SyncBehavior $behavior): SyncResult;
+    public function sync(iterable $operations, Context $context, SyncBehavior $behavior): SyncResult;
```

Comment:
BC 🤔

Reply:
Will this change never make it into Shopware or do you mean, I should target a different branch?

Reply:
We have a new branch - major - for the next well major release, where we can implement breaking changes. But the branch is highly unstable and will be rebased/force pushed.